xarray/__init__.py

```diff
@@ -75,6 +76,7 @@
     "full_like",
     "get_options",
     "infer_freq",
+    "initialize_zarr",
```

Comment:
Do we want `xarray.zarr.initialize_zarr` or `xarray.backends.initialize_zarr` instead?

Reply:
Is there a case for `Dataset.to_zarr_initialize`? It's a bit awkward, but it does avoid another top-level method, which is harder to find for non-expert users.